spirv_glsl.hpp

```diff
@@ -297,6 +297,9 @@ class CompilerGLSL : public Compiler
 		float_formatter = formatter;
 	}
 
+	// Returns the macro name corresponding to constant id
+	std::string constant_value_macro_name(uint32_t id);
```

Comment:
Where is this implemented?

Reply:
This is an existing method, I just moved it from protected to public so that external user (MoltenVK) can query the macro name.

Reply:
I can make it const in the next round.

Reply:
Now the method is const.